docs/src/domains/cdn/_index.md

```diff
@@ -25,29 +25,29 @@ For more information, see you DNS provider's official documentation.
 
 Note that [`CNAME` records can't point to apex domains](../steps/dns.md),
 but most CDN providers offer workarounds.
-For example, Fastly offers [Anycast options](./fastly.md#3-handle-apex-domains) 
+For example, Fastly offers [Anycast options](./fastly.md#3-handle-apex-domains)
```

Comment:
❓ Not sure why there are changes from the CDN docs PR in the history of this PR? 🤔